types/linkifyjs/tsconfig.json

```
@@ -13,5 +13,5 @@
         "forceConsistentCasingInFileNames": true,
         "jsx": "preserve"
     },
-    "files": ["index.d.ts", "html.d.ts", "react.d.ts", "linkifyjs-tests.tsx"]
+    "files": ["index.d.ts", "html.d.ts", "react.d.ts", "linkifyjs-tests.tsx", "element.d.ts", "string.d.ts"]
```

We probably should have done this earlier, but can you change this to into a sorted, vertical list?

```suggestion
    "files": [
      "element.d.ts",
      "html.d.ts",
      "index.d.ts",
      "linkifyjs-tests.tsx",
      "react.d.ts",
      "string.d.ts",
    ]
```